appeals/api/src/server/tests/data.js

```diff
@@ -686,13 +687,27 @@ const baseExpectedAppellantCaseResponse = (appeal) => ({
 	}
 });
 
+const document = {
+	folderId: 1,
+	path: 'path/to/document/folder',
```

Comment:
There is a routine which splits the document path to extract stage (eg. `lpa_questionnaire`) and doc type (eg: `appealStatement`)... Maybe would be better to have a mock path along those lines (eg .`lpa_questionnaire/emergingPlans`).

Reply:
Yeah could do that, I was trying to have a generic test document object that could apply to all documents but we could make it a function and pass in the path as a param.

Otherwise we'll need to create separate test data for each document, which was the path I was going down and then created this to avoid that!

Reply:
no worries, not a big deal - just worried that some validator may impact on tests but sounds all good for now!